tpl/bootstrap.php

```diff
@@ -91,7 +91,6 @@
 endif;
 if ($isDark):
 ?> class="dark-theme"<?php
-endif;
```

Comment:
I assume we want to use the password modal in the regular bootstrap theme as well. I'll revert this.

Reply:
Ugh, yes, I don't know why this came in. I ran the `phpunit` tests to regenerate the SRI hashes.

The unit tests must have changed that… :thinking: